### PR TITLE
CLDR-16320 hi_Latn: drop compact numbers

### PR DESCRIPTION
**Pull request by srl295**
- keep draft status but change to triple up arrow

CLDR-16320

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: common/main/hi_Latn.xml [hunk 1]
```diff
@@ -5361,58 +5361,58 @@ annotations.
 			</decimalFormatLength>
 			<decimalFormatLength type="long">
 				<decimalFormat>
```
Comment (btangmu) [inline common/main/hi_Latn.xml:5363]: do these paths under "long" count as "compact"? I imagine they do given the ticket name... I'm ignorant about what "compact" means here...

**Review by btangmu — APPROVED** [review]
looks fine except for my ignorance about how "compact" relates to "long" and "short" here

File: common/main/hi_Latn.xml (hunk 1 above)
Comment (srl295) [reply common/main/hi_Latn.xml:5363]: that's a good question.  I think they are still compact in that they aren't spelled out.   (`123 thousand`,  not `one hundred twenty three thousand`) .  Maybe get @macchiati  and @AEApple comment?

Comment (AEApple) [reply common/main/hi_Latn.xml:5363]: Yes, it's that they aren't spelled out as a number so 1 million would be the compact number form vs. 1,000,000.

**Pull request merged**